packages/@aws-cdk/aws-apigateway/README.md

```diff
@@ -565,7 +565,7 @@ books.addMethod('GET', new apigateway.HttpIntegration('http://amazon.com'), {
 });
 ```
 
-You can find a full working example [here](test/authorizers/integ.request-authorizer.ts).
+You can find a full working example [here](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-apigateway/test/authorizers/integ.token-authorizer.ts).
```

Comment:
We should switch to a `.lit.ts` format so that this gets expanded and translated when published to package managers.

Example: 
source: https://github.com/aws/aws-cdk/blob/3800830609e67d6ffc4e18dd5d61bb67d90a066b/packages/%40aws-cdk/aws-servicediscovery/README.md#private-dns-namespace-example
rendered on api ref: https://docs.aws.amazon.com/cdk/api/latest/docs/aws-servicediscovery-readme.html#private-dns-namespace-example
rendered on python ref: https://docs.aws.amazon.com/cdk/api/latest/python/aws_cdk.aws_servicediscovery.README.html#private-dns-namespace-example

Reply:
I see. (Also see I linked the wrong example.)